pom.xml

```diff
@@ -1718,9 +1718,8 @@
                 <plugin>
                     <groupId>org.apache.maven.plugins</groupId>
                     <artifactId>maven-surefire-plugin</artifactId>
-                    <version>3.0.0-M5</version>
+                    <version>3.0.0-M6</version>
```

Comment:
Upgrading surefire to fix [SUREFIRE-1926](https://issues.apache.org/jira/browse/SUREFIRE-1926).